index.js

```diff
@@ -197,11 +197,17 @@ Plaid.Client.prototype.deleteInfoUser =
 
 
 // exchangeToken
-Plaid.Client.prototype.exchangeToken = function(public_token, callback) {
+Plaid.Client.prototype.exchangeToken =
+  function(public_token, account_id, callback) {
+  if (typeof account_id === 'function') {
+    callback = account_id;
+    account_id = null;
+  }
```

Comment:
I don't entirely disagree, @davidchambers. As it stands now, the library uses argument juggling in multiple places. I'll open an issue to discuss removing this functionality in a future minor release.